### DISCUSSION TITLE
执行show frontends执行太慢

### DISCUSSION THREAD

**Discussion by ttwode** [apache/doris#12153] [A - General / Q&A]
1.1.1arm版本部署完成后，3个FOLLOWER，执行show frontends; 30多秒。这个啥问题啊，太慢了。

**Comment by ttwode** [comment]
已解决

**Reply by dataalive** [reply]
具体什么问题，怎么解决的可以分享下吗？

**Reply by ttwode** [reply]
就是部署完成后，执行 show frontends 命令，结果返回很慢要30多秒，执行show backends 很快。最后检查发现是文件 /etc/resolv.conf 中配置了dns服务器，把 /etc/resolv.conf 配置的nameserver注释掉，show frontends秒查。

**Reply by xiaoyunjie** [reply]
感谢，同样遇到类似问题

**Reply by hcxx-x** [reply]
nameserver 注释掉之后无法访问网络了 怎么办

**Comment by ttwode** [comment]
邮件已收到

**Comment by ttwode** [comment]
邮件已收到

**Comment by ttwode** [comment]
邮件已收到
